src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -198,7 +198,7 @@ public Glyph Glyph
 
             public string GetFinalSymbolName(string replacementText)
             {
-                if (_isRenamingAttributePrefix)
+                if (_isRenamingAttributePrefix && !TryGetWithoutAttributeSuffix(replacementText, out var _))
```

Comment:
if TryGetWithoutAttributeSuffix only used here.  can it be just HasAttributeSuffix? rather than return something in out param and nobody use it?